react-native-fbsdkcore/js/FBSDKGraphRequest.ios.js

```diff
@@ -17,69 +17,71 @@
 * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 *
-* @providesModule FBSDKGraphRequest
 * @flow
 */
 
 'use strict';
 
-/*
+var FBSDKGraphRequestManager = require('./FBSDKGraphRequestManager.ios.js');
```

Comment:
Yeaaahhhhh. I've thought about this a bit, but in the end decided to keep them. Chiefly for the reason that it's nice to have a clear identifier when you're doing

```js
var {FBSDKBlaBla} = require('react-native-fbsdkcore');
```

rather than just `BlaBla`, because `BlaBla` could easily be a generic word like `LoginManager` or `LikeButton`.